File: kreyes-gourmet-cuisine/src/com/kreyes/gourmet/cuisine/entities/MeasureUnit.java
package com.kreyes.gourmet.cuisine.entities;

import java.util.Objects;

public class MeasureUnit {
    private Long id;
    private String name;

    public MeasureUnit() {
    }

    public MeasureUnit(Long id) {
        this.id = id;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        MeasureUnit measureUnit = (MeasureUnit) o;
        return Objects.equals(id, measureUnit.id);
    }

    @Override
    public int hashCode() {
        return Objects.hash(id);
    }
}

File: millord-restaurante-app/src/entities/MeasureUnit.ts
import { Md5 } from 'ts-md5/dist/md5'



export class MeasureUnit {
  private id: string;
  private name: string;
  constructor() { }

  public getId(): string {
    return this.id;
  }

  public setId(id: string): void {
    this.id = id;
  }

  public getName(): string {
    return this.name;
  }

  public setName(name: string): void {
    this.name = name;
  }
  public hashCode() {
    return Md5.hashStr(this.id)
  }



}